webapp/mixins/PostMutationHelpers.js

```diff
@@ -0,0 +1,39 @@
+import gql from 'graphql-tag'
+
+export default {
+  methods: {
+    async deletePostCallback(postDisplayType = 'list') {
```

Comment:
As I said, try to avoid mixins. I guess this one might be replaced in a future refactoring. It's OK for now.